maven/pom.xml

```diff
@@ -31,12 +31,16 @@
      
   <name>${project.groupId}:${project.artifactId}</name>
 
+  <properties>
+    <maven.plugin.version>3.6.1</maven.plugin.version>
```

Comment:
I don't see any usage of this propertie, is it correct?

Reply:
I planned to use it, but it turned out, that all maven dependencies have different versions. Will remove it.